templates/50_singularity.config.erb

```diff
@@ -5,10 +5,11 @@ SINGULARITY = <%= @singularity_path %>
 SINGULARITY_JOB = <%= @force_singularity_jobs %>
 
 # Forces all jobs to use the given image.
-SINGULARITY_IMAGE_EXPR = "<%= @singularity_image_expr %>"
+SINGULARITY_IMAGE_EXPR = <%= @singularity_image_expr %>
 
-# Paths to bind via user bind control.
+# Paths to bind via user bind control. 
```

Comment:
It seems your editor like to keep spaces at the end of lines ;)

Reply:
This came back when I issued `git revert` to revert your commit which broke `SINGULARITY_IMAGE_EXPR` ;-). 
But I can remove the space again if you like.